core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/collection/MembershipExpression.kt

```diff
@@ -0,0 +1,15 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.collection
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.operator.InfixOperator
+import ch.ergon.dope.validtype.ArrayType
+import ch.ergon.dope.validtype.BooleanType
+import ch.ergon.dope.validtype.ValidType
+
+sealed class MembershipExpression<T : ValidType>(
```

Comment:
do we really need this expression? I think we save much code with it. Just inherit directly from _TypeExpression< BooleanType >, InfixOperator(value, symbol, collection)_ right?

Reply:
i removed it, done